### DISCUSSION TITLE
docs/troubleshooting/

### DISCUSSION THREAD

**Discussion by giscus** [fl0zone/fl0-docs#23] [Comments]
# docs/troubleshooting/

Unfortunately, sometimes problems happen! This page outlines some of the most common errors and provides advice on solving them.

https://docs.fl0.com/docs/troubleshooting/

**Comment by enosefelix** [comment]
I'm trying to load .env variable DATABASEURL in my prisma, but I keep getting this error:

Error: P1013: The provided database string is invalid. empty host in database URL. Please refer to the documentation in https://www.prisma.io/docs/reference/database-reference/connection-urls for constructing a correct connection string. In some cases, certain characters must be escaped. Please check the string for any illegal characters.
